### ISSUE TITLE
AdBlockPlus

### ISSUE THREAD

**Issue by Dr-Flay** [smicallef/spiderfoot#313]
What is the theory behind using the default Easylist ?
It is mostly focused on adverts and trackers.
Wouldn't it be better to pick something from Steven Black's selection based on use case such as malware or spam ?
https://github.com/StevenBlack/hosts

**Comment by bcoles** [comment]
The theory behind using the AdBlock easylist for the `sfp_adblock` module is to test if external/internally linked pages would be blocked by AdBlock Plus.

Using StevenBlack lists should be implemented as a separate module. There's probably also some crossover between these lists and the existing modules. `grep -r MALICIOUS_ modules`

**Comment by Dr-Flay** [comment]
Yeah I get that it is checking for advert domains, but my point is why is that useful when a malvertising and malware list would seem more useful and in line with the point of SpiderFoot.
I don't get why all advertising is contemplated when evaluating vulnerability. Surely that is just going to introduce more false positives the user has to work out what to ignore.

**Comment by bcoles** [comment]
> why is that useful

It's nice to have options. The module is pretty old (2014), and was added long before the newer modules which search reputation systems, such as Talos, Binary Edge, RiskIQ, and Abuse IP DB, + a bunch of others which were added in 2018/2019.

> why all advertising is contemplated when evaluating vulnerability

The AdBlock module returns `["URL_ADBLOCKED_INTERNAL", "URL_ADBLOCKED_EXTERNAL"]` data types, rather than `VULNERABILITY` or `MALICIOUS_*` data types.

Spiderfoot offers granular configuration options. Users can turn off the Adblock module, or filter out the `URL_ADBLOCKED_*` events, should they wish.

**Comment by bcoles** [comment]
This has been added in #1315 which uses the adware + malware list by default.
